packages/generator/src/generator-options.ts

```diff
@@ -200,7 +221,8 @@ export const generatorOptionsCli: KeysToOptions = {
       'By default, when generating package.json file, the generator will set a version by using the generator version. It can also be set to a specific version.',
     type: 'string',
     deprecated:
-      "Since v2.6.0. Use the 'include' option to add your own package.json file instead."
+      "Since v2.6.0. Use the 'include' option to add your own package.json file instead.",
+    replacedBy: 'packageVersion'
```

Comment:
[q] This I do not understang. The replace is `packageVersion` and in the sentence above it is mentioned to use `include`.

Reply:
Yes, this is confusing. Technically, the replacement for this is `packageVersion`. We moved this to a hidden parameter `packageVersion` because it was useful to us for the pregenerated clients, but most likely not relevant for users.

However, I now see the issue that if you do batch processing, `include` is not an adequate replacement, because you would always add the same package.json for all services...